.travis.yml

```diff
@@ -10,7 +10,7 @@ matrix:
   include:
     - os: linux
       compiler: clang
-      env: BTYPE=RelWithDebInfo WRAP=on  DOXY=on  NPROC=1 DEPLOY=yes
+      env: BTYPE=RelWithDebInfo WRAP=off  DOXY=on  NPROC=1 DEPLOY=yes
```

Comment:
Wrapping is not being tested on travis; I thought the intent of this PR was to enable testing of wrapping on travis?